app/src/main/java/org/oppia/app/player/exploration/ExplorationContentViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.app.player.exploration
+
+/** [ViewModel] for [StatePlayerRecyclerViewAssembler] items. */
+class ExplorationContentViewModel(hasBlueBackground: Boolean, isCenterAligned: Boolean) {
```

Comment:
The naming is incorrect: This is actually not a view model, this is just a simple class.

Also, why do we need this at exploration level and not at `StateFragment` level? Because the the content view is a part of recyclerview which is also a part of StateFragment, so this class should be connected to that. Or is there some other logic related to this?

Reply:
Renamed to PlayerRecyclerViewAssemblerViewModel